qiita_pet/templates/analysis_results.html

```diff
@@ -1,8 +1,15 @@
 {% extends sitebase.html%}
 
 {%block head%}
-
-
+<link rel="stylesheet" href="/static/vendor/css/select2.min.css" type="text/css">
+<script type="text/javascript" src="/static/vendor/js/select2.min.js"></script>
+<script type="text/javascript" src="/static/js/sharing.js"></script>
+<script type="text/javascript">
+$(document).ready(function () {
+  current_id = {{analysis_id}};
+  update_share('analysis');
```

Comment:
No, I'm saying that the calls on line 21 and 25 of sharing.js have no idea what the current_id is if it is not a global variable, so this suggestion will not work.